Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.yml

```
@@ -0,0 +1,191 @@
+category: Forensics & Malware Analysis
+commonfields:
+  id: WildFire-Reports
+  version: -1
+configuration:
+- defaultvalue: https://wildfire.paloaltonetworks.com/publicapi
+  display: Server base URL (e.g. https://192.168.0.1/publicapi)
+  name: server
+  required: true
+  type: 0
+- display: API Key
+  name: token
+  required: true
+  type: 4
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: B - Usually reliable
+  display: Source Reliability
+  name: integrationReliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Return warning entry for unsupported file types
+  name: suppress_file_type_error
+  required: false
+  type: 8
+description: generates a WildFire PDF report.
+display: WildFire Reports
+name: WildFire-Reports
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: MD5 hash to check.
+      isArray: true
+      name: md5
+      required: false
+      secret: false
+    - default: false
+      description: SHA256 hash to check
+      isArray: true
+      name: sha256
+      required: false
+      secret: false
+    - default: false
+      description: Deprecated - Use the sha256 argument instead.
+      isArray: true
+      name: hash
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: Receive extended information from WildFire.
+      isArray: false
+      name: verbose
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves results for a file hash using WildFire.
+    execution: false
+    name: wildfire-report
```

let's have a unique name, so it won't collide with the current integration.
```suggestion
    name: wildfire-get-report
```